internal/rand.go

```diff
@@ -0,0 +1,12 @@
+package internal
+
+import (
+	"math/rand"
+	"time"
+)
+
+// RandUint32 returns random uint32 value [0, max).
+func RandUint32(max uint32) uint32 {
+	rand.Seed(time.Now().UnixNano())
+	return rand.Uint32() % max
+}
```

Comment:
Why do you seed every time? I think, single `init()` is sufficient. Also, helps with reproducibility.

BTW, what about putting this in `internal/random`? We could benefit from also having `random.String(n)` and `random.Bytes(n)`. We already have similar thing in neo-go https://github.com/nspcc-dev/neo-go/blob/master/internal/random/random_util.go

Reply:
Agree, all done.